engine/execution/testutil/fixtures.go

```diff
@@ -179,26 +179,42 @@ func CreateAccountsWithSimpleAddresses(
 
 	var accounts []flow.Address
 
-	script := []byte(`
-	  transaction(publicKey: [UInt8]) {
-	    prepare(signer: AuthAccount) {
-	  	  let acct = AuthAccount(payer: signer)
-	  	  acct.addPublicKey(publicKey)
-	    }
-	  }
-	`)
+	scriptTemplate := `
+        transaction(publicKey: [UInt8]) {
```

Comment:
We've rolled back the removal of the deprecated key management API, but it's still good to start using the new one 👍